source/dotnet/Library/AdaptiveCards.Rendering.Html/AdaptiveCardRenderer.cs

```diff
@@ -378,6 +378,13 @@ protected static void AddActions(HtmlTag uiContainer, IList<AdaptiveAction> acti
                     }
                 }
 
+                // If the number of actions is bigger than maxActions, then log warning for it
+                if (actions.Count > actionsConfig.MaxActions)
+                {
+                    context.Warnings.Add(new AdaptiveWarning((int)AdaptiveWarning.WarningStatusCode.MaxActionsExceeded,
+                        "Some actions were not rendered due to exceeding the maximum number of actions allowed"));
```

Comment:
>"Some actions were not rendered due to exceeding the maximum number of actions allowed" [](start = 24, length = 87)

[nit] Since we use this string in two places consider defining it as a const.